ResearchKit/ActiveTasks/ORKAudioContentView.m

```diff
@@ -47,7 +47,7 @@ @interface ORKAudioGraphView : UIView
 @property (nonatomic, strong) UIColor *keyColor;
 @property (nonatomic, strong) UIColor *alertColor;
 
-@property (nonatomic, copy) NSArray *values;
+@property (nonatomic, strong) NSArray *values;
```

Comment:
My initial patch changed it from `strong` to `copy, then @DanKeen requested changing it back to `strong`. See his comment above.